### PR TITLE
Add Terms of Service and update contact details in footer

### PR DESCRIPTION
**Pull request by josh-**

### CODE REVIEW

**Comment by josh-** [comment]
@dsykesturner combined the refunds policy into a new "Terms of Service" – let me know what you think about this idea 😄

File: terms.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+{::options parse_block_html="true" /}
+<div class="col-lg-8 col-md-12 col-xs-12">
+
+UNIHACK INCORPORATED is an Australian-owned and non-for-profit company. All payments will be invoiced in Australian Dollars unless otherwise agreed upon in writing.
+
+### UNIHACK Competitor Refund Policy
+
+Unfortunately we cannot offer refunds for any competitor tickets, however you are able transfer your ticket to another student willing to compete.
+
+Our [Facebook Group](https://facebook.com/groups/unihackers/) is the recommended medium to find other students looking for tickets.
```
[the remark this answers is not in the document]
Comment (dsykesturner) [reply terms.md]: Oh haha I don't mind the facebook group. We don't answer people's questions or anything do we?

[the remark this answers is not in the document]
Comment (josh-) [reply terms.md]: Ooohhh guess it depends who's in there 😝

[the remark this answers is not in the document]
Comment (josh-) [reply terms.md]: Sounds good though!

**Pull request merged**